shared_model/cryptography/blob.hpp

```diff
@@ -18,20 +18,18 @@
 #ifndef IROHA_SHARED_MODEL_BLOB_HPP
 #define IROHA_SHARED_MODEL_BLOB_HPP
 
-#include <iomanip>
-#include <sstream>
+#include <string>
 #include <vector>
-#include "common/byteutils.hpp"
+
 #include "interfaces/base/model_primitive.hpp"
 #include "utils/lazy_initializer.hpp"
-#include "utils/string_builder.hpp"
 #include "utils/swig_keyword_hider.hpp"
 
 namespace shared_model {
   namespace crypto {
 
     class Blob;
-    static inline std::string toBinaryString(const Blob &b);
+    inline std::string toBinaryString(const Blob &b);
```

Comment:
If we have only a declaration here, why make it inline?

Reply:
https://stackoverflow.com/a/9268589/1953079

Declaration should contain inline.